lego/apps/users/models.py

```diff
@@ -299,8 +300,17 @@ def email_address(self):
     def profile_picture(self, value):
         self.picture = value
 
+    @PermissionsMixin.is_abakus_member.setter
+    def is_abakus_member(self, add_to_group):
+        if self.is_verified_student():
+            abakus_group = AbakusGroup.objects.get(name=constants.MEMBER_GROUP)
+            if add_to_group:
+                abakus_group.add_user(self)
+            else:
+                abakus_group.remove_user(self)
+
     def is_verified_student(self):
-        return self.student_username is not None
+        return self.student_username is not None and self.grade is not None
```

Comment:
shouldn't these be equivalent? to me it seems like if we have to check both of these here, we're not doing validation properly somewhere else - i.e. should you be able to be in a grade without having a student username?

Reply:
They should, but the fixtures are not - will fix that and revert this. But we have to take this into consideration when adding bumping of grades, since we will have to remove the student_username - since this check should be false when they are alumni.